Automattic/wp-calypso
# Ecommerce onboarding: Ensure new ecommerce sites are set to coming-soon

#### Proposed Changes

Previously, running through the new tailored ecommerce flow created the new site set to public. This change makes new ecommerce sites so they're set to coming-soon instead.

#### Testing Instructions



* Apply this branch or use the Calypso Live link.
* Visit `/setup/ecommerce?flags=signup/tailored-ecommerce` and create a new ecommerce site.
* Verify that the newly created site _is not_ set to Public.

#### Pre-merge Checklist



- [NA] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [NA] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [NA] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [NA] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [NA] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?


Fixes #70470
